curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-040.md

```diff
@@ -13,10 +13,16 @@ Change the `width` value to `49%` for each class to see what happens.
 
```

Comment:
First paragraph typo: `...placing them on a separate lines in...` , `... make each p's width...` - best to change to `...make each p elements width...` for i18n